### PR TITLE
Bump debian pg_cron to 1.5.1

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+pgcron (1.5.1.citus-1) stable; urgency=low
+
+  * Official 1.5.1 release of Pg Cron
```
Comment (hanefi) [inline debian/changelog]: This is a weird way to write an abbreviation for PostgreSQL. How about dropping the name altogether, or capitalizing PG part?

```suggestion
  * Official 1.5.1 release
```
or
```suggestion
  * Official 1.5.1 release of PG Cron
```

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+ -- Gurkan Indibay <marco@citusdata.com>  Thu, 4 Apr 2022 10:55:45 +0000
```
Comment (hanefi) [inline debian/changelog]: The name is yours, but email belongs to Marco. This feels weird.

**Review by hanefi — CHANGES_REQUESTED** [review]

File: debian/changelog (hunk 1 above)
Comment (gurkanindibay) [reply debian/changelog]: Fixed with the second alternative

File: debian/changelog (hunk 2 above)
Comment (gurkanindibay) [reply debian/changelog]: Sorry for the confusion. Fixed email address

**Review by hanefi — APPROVED** [review]

**Pull request merged**